src/test/end-to-end/paths/loginUser.js

```diff
@@ -1,7 +1,7 @@
 const testConfig = require('../../config');
-const createCaseConfig = require('src/test/end-to-end/pages/createCase/createCaseConfig');
+
 Feature('`Verify login smoke scenario');
 
 Scenario('@smoke login to the manage case application', async ({ I }) => {
```

Comment:
Could you extend smoke test till end of the case submission. Initially we discussed about the use-case of Login and logout of the application, but we need to extend the steps till end of the case submission.

Reply:
I don't think the smoke tests should have the entire journey, let's discuss this.